specification/databricks/resource-manager/Microsoft.Databricks/preview/2025-08-01-preview/databricks.json

```diff
@@ -0,0 +1,1989 @@
+      "properties": {
+        "automaticClusterUpdate": {
+          "type": "string",
+          "description": "Status of automated cluster updates feature.",
+          "$ref": "#/definitions/AutomaticClusterUpdateDefinition"
+        },
+        "complianceSecurityProfile": {
+          "type": "object",
+          "description": "Status of Compliance Security Profile feature.",
+          "$ref": "#/definitions/ComplianceSecurityProfileDefinition"
+        },
+        "enhancedSecurityMonitoring": {
+          "type": "object",
+          "description": "Status of Enhanced Security Monitoring feature.",
+          "$ref": "#/definitions/EnhancedSecurityMonitoringDefinition"
+        }
+      }
+    },
+    "AutomaticClusterUpdateDefinition": {
+      "type": "object",
+      "description": "Status of automated cluster updates feature.",
+      "properties": {
+        "value": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "AutomaticClusterUpdateValue"
+          }
+        }
+      }
+    },
+    "EnhancedSecurityMonitoringDefinition": {
+      "type": "object",
+      "description": "Status of Enhanced Security Monitoring feature.",
+      "properties": {
+        "value": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "EnhancedSecurityMonitoringValue"
+          }
+        }
+      }
+    },
+    "ComplianceSecurityProfileDefinition": {
+      "type": "object",
+      "description": "Status of Compliance Security Profile feature.",
+      "properties": {
+        "complianceStandards": {
+          "description": "Compliance standards associated with the workspace.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ComplianceStandardDefinition"
+          }
+        },
+        "value": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "ComplianceSecurityProfileValue"
+          }
+        }
+      }
+    },
+    "ComplianceStandardDefinition": {
+      "description": "Compliance standard that can be associated with a workspace.",
+      "type": "string",
+      "enum": [
+        "NONE",
+        "HIPAA",
+        "PCI_DSS",
+        "CYBER_ESSENTIAL_PLUS",
+        "FEDRAMP_HIGH",
+        "CANADA_PROTECTED_B",
+        "IRAP_PROTECTED",
+        "ISMAP",
+        "HITRUST",
+        "K_FSI"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "ComplianceStandard"
+      }
+    },
+    "WorkspaceCustomParameterType": {
+      "description": "Provisioning status of the workspace.",
```

Comment:
>description": "Provisioning status of the workspace.", [](http://example.com/codeflow?start=7&length=54)

is this description accurate for WorkspaceCustomParameterType? It doesn't look accurate